tasks/main.yml

```diff
@@ -0,0 +1,28 @@
+- name: install satellite
+  package:
+    name: "{{ satellite_packages }}"
+    state: present
+  become: true
+
+- name: upload answers file
+  template:
+    src: satellite-answers.yaml
+    dest: "{{ satellite_answers_file_destination }}"
+    owner: root
+    group: root
+    mode: 0600
+  become: true
+  notify: run satellite installer
+
+- name: upload scenario file
+  template:
+    src: satellite.yaml
+    dest: "{{ satellite_scenario_location }}"
+    owner: root
+    group: root
+    mode: 0600
+  become: true
+  notify: run satellite installer
```

Comment:
Would it be possible to combine the previous task and this one such that the handler needs to be called only once or is there a specific reason it's separated like this?